infrastructure/lambda/template.yaml

```diff
@@ -191,6 +187,10 @@ Globals:
         }
 
 Mappings:
+  StaticVariables:
+    Urls:
+      SupportManualURL: "https://team-manual.account.gov.uk/teams/CRI-Orange-team/supporting-cri-orange/address-credential-issuer-runbook"
```

Comment:
Should this include the anchor link to the relevant section? Indifferent either way, just thought the question was worth asking as it could move down the page at some point in the future

Reply:
Yeah this support page unfortunately doesn't have nice anchors splitting the alarms into FE and BE so the anchor would just go to the top of the page

Reply:
Yeah, fair enough